csunplugged/topics/views.py

```diff
@@ -345,6 +350,17 @@ def get_queryset(self):
         """
         return GlossaryTerm.objects.order_by("term")
 
+    def get_context_data(self):
+        """Get context data for template rendering."""
+        return {
+            "glossary_terms": GlossaryTerm.objects.filter(
+                Q(languages__contains=[get_language()])
+            ).order_by("term_en"),
+            "untranslated_glossary_terms": GlossaryTerm.objects.filter(
```

Comment:
Why is `untranslated_glossary_terms` added to the context?

Reply:
As part of the UI prototype, the glossary page showed translated terms, followed by untranslated terms in English, under a warning banner. 

Something similar will be required for the final UI, so removing it would only be temporary. Leaving it here doesn't change the user experience aside from a very minor increase in response time.

Thoughts?

Reply:
Happy to leave it here.